src/openms/source/QC/FragmentMassError.cpp

```diff
@@ -49,8 +50,31 @@
 
 namespace OpenMS
 {
-  void FragmentMassError::compute(FeatureMap& fmap, const MSExperiment& exp, const double tolerance, const ToleranceUnit tolerance_unit)
+  const std::string FragmentMassError::names_of_toleranceUnit[] = {"PPM", "DA", "AUTO"};
```

Comment:
can you make the values lower case?! This is how its usually done in OpenMS' parameters.